<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>810</width>
      <height>276</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="icon">
      <iconset resource="pics.qrc">
       <normaloff>:/images/cartoon1.ico</normaloff>:/images/cartoon1.ico</iconset>
     </attribute>
     <attribute name="title">
      <string>多项式拟合</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>190</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>190</width>
            <height>190</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="pics.qrc">:/images/1.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonMul">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>产生测试数据</string>
          </property>
          <property name="icon">
           <iconset resource="pics.qrc">
            <normaloff>:/images/cartoon3.ico</normaloff>:/images/cartoon3.ico</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>190</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>190</width>
            <height>190</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="pics.qrc">:/images/2.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonLine">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>多项式拟合</string>
          </property>
          <property name="icon">
           <iconset resource="pics.qrc">
            <normaloff>:/images/cartoon1.ico</normaloff>:/images/cartoon1.ico</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>190</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>190</width>
            <height>190</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="pics.qrc">:/images/3.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonLine_bayes">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>贝叶斯曲线拟合</string>
          </property>
          <property name="icon">
           <iconset resource="pics.qrc">
            <normaloff>:/images/cartoon1.ico</normaloff>:/images/cartoon1.ico</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>190</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>190</width>
            <height>190</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="pics.qrc">:/images/3.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_probability">
          <property name="maximumSize">
           <size>
            <width>190</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>概率统计</string>
          </property>
          <property name="icon">
           <iconset resource="pics.qrc">
            <normaloff>:/images/cartoon1.ico</normaloff>:/images/cartoon1.ico</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="icon">
      <iconset resource="pics.qrc">
       <normaloff>:/images/cartoon1.ico</normaloff>:/images/cartoon1.ico</iconset>
     </attribute>
     <attribute name="title">
      <string>贝叶斯</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
